### PR TITLE
Jetpack Scan: Reorganize sections in tab nav

### PR DESCRIPTION
**Pull request by monsieur-z**
#### Changes proposed in this Pull Request

Reorganize the _Scan_ section in wordpress.com and cloud.jetpack.com using a tab navigation, as part of the Jetpack.com alignment initiative.

#### Implementations notes

* Cards in the threat list are different than the ones in the mockups (see screenshots below). This should be a different PR.
* The `classname.is_jetpackcom` pattern doesn't seem pretty solid moving forwards. At least in the scan section we could try to avoid duplication of styles.

#### Testing instructions

1. Download the PR and run Calypso. Add the `flags=jetpack/features-section` query param.
2. Check that the layout and structure of the _Scan_ section matches the mockups (screenshots below)
3. Check that the Scanner and History functionalities still work
4. Check that the layout looks good on mobile as well
5. Check that the _Scan_ section is not available (via the menu or the routes directly) when you remove the query param
6. Run Jetpack cloud
7. Check on desktop and mobile that the layout and structure of the _Scan_ section are similar to Calypso (the _Jetpack Scan_ heading should not be present)
8. Check that the Scanner and History functionalities still work

#### Snapshots

**Scanner**
Screen Shot 2020-06-12 at 9 49 38 AM

**History**
Screen Shot 2020-06-12 at 9 49 51 AM

### CODE REVIEW

File: client/my-sites/scan/navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export default function ScanNavigation( { section }: Props ) {
+	const site = useSelector( getSelectedSite );
+	const translate = useTranslate();
+
+	if ( ! site ) {
+		return <SectionNav />;
+	}
+
+	const strings = {
+		scanner: translate( 'Scanner', { context: 'scan screen' } ),
```
Comment (a8ci18n) [inline client/my-sites/scan/navigation.tsx:29]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Scanner' )` *ES Score: 15*


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.

File: client/my-sites/scan/navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		history: translate( 'History', { context: 'scan screen' } ),
```
Comment (a8ci18n) [inline client/my-sites/scan/navigation.tsx:30]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **87** times:
`translate( 'History' )` *ES Score: 13*


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.

File: client/my-sites/scan/main.tsx [hunk 3]
```diff
@@ -231,40 +233,29 @@ class ScanPage extends Component< Props > {
...
 		const { siteId } = this.props;
+		const isJetpackPlatform = isJetpackCloud();
+
 		if ( ! siteId ) {
 			return;
 		}
-		const isJetpackCom = !! config( 'env_id' ).includes( 'jetpack-cloud' );
+
 		return (
 			<Main
```
Comment (mattgawarecki) [inline client/my-sites/scan/main.tsx:245]: To match Backup and some other Calypso pages, add `wideLayout={ ! isJetpackCloud() }`. I'm not sure this is the logic we'll stick with long-term, but for now it appears to work.

Comment (monsieur-z) [reply client/my-sites/scan/main.tsx:245]: Thks! Updated.

**Comment by monsieur-z** [comment]
> This looks good with one minor exception:
> 
> Screen Shot 2020-06-12 at 12 11 48 PM
> 
> Screen Shot 2020-06-12 at 12 11 54 PM
> 
> In Jetpack Cloud, the navigation bar should not be visible on the scan or history pages.

For the sake of grouping discussions in this thread, the navigation bar is now present in Jetpack cloud as well.

**Pull request merged**